CITATION.cff

```diff
@@ -0,0 +1,32 @@
+cff-version: 1.2.0
+title: NOMAD Simulations Schema Plugin
```

Comment:
Only "NOMAD Simulations"

I prefer to separate concepts like schema or plugin from the repo itself (although it works being compliant with it)